BenchmarkComponent/BenchmarkComponent.idl

```diff
@@ -32,4 +32,24 @@ namespace BenchmarkComponent
         Boolean DefaultBoolProperty{ get; set; };
         Double DefaultDoubleProperty{ get; set; };
     }
+
+    [default_interface]
+    runtimeclass WrappedClass

```

Comment:
Is WrappedClass used in the benchmarks?

Reply:
Not yet but it will be used in future when we benchmark other marshaling routines